content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -203,8 +203,10 @@ running such processes via a DaemonSet:
 - Ability to monitor and manage logs for daemons in the same way as applications.
 - Same config language and tools (e.g. Pod templates, `kubectl`) for daemons and applications.
 - Running daemons in containers with resource limits increases isolation between daemons from app
-  containers.  However, this can also be accomplished by running the daemons in a container but not in a Pod
-  (e.g. start directly via Docker).
+  containers.  However, this can also be accomplished by running the daemons in a container but not in a Pod.
+   {{< warning >}}
+    code samples potentially breaking. Test code samples on a cluster with a different runtime and on a cluster with cri-dockerd and Docker.
+   {{< /warning >}}
```

Comment:
This warning doesn't look right. What's your intention here @889-dj ?

Reply:
It looks like you are copying @shannonxtreme's instruction to _authors_. The idea is to follow that instruction, rather than paste it in as-is into the documentation that end users would read.

Reply:
I m sorry @sftim I should have asked for help could you please suggest what can be improved as I am not aware of the term cri-dockered and was learning about that through google
also as suggested by Sean I was going through the related changes and documentation and trying to understand the concepts

Reply:
@889-dj It seems like you're seeking training and education. That's understandable, but unfortunately it's beyond the scope of what SIG Docs can provide, especially in a PR review.

Reply:
> @889-dj It seems like you're seeking training and education. That's understandable, but unfortunately it's beyond the scope of what SIG Docs can provide, especially in a PR review.

I understand sure I will learn more about it and will be looking forward to make more contribution 😃